codec-server/go.mod

```diff
@@ -4,9 +4,9 @@ go 1.21.1
 
 require (
 	github.com/golang/snappy v0.0.4
-	github.com/google/uuid v1.5.0
-	go.temporal.io/api v1.26.0
-	go.temporal.io/sdk v1.25.1
+	github.com/google/uuid v1.6.0
+	go.temporal.io/api v1.43.0
+	go.temporal.io/sdk v1.32.1
 	go.temporal.io/server v1.22.4
```

Comment:
We should update the server too here.

Reply:
@bergundy I noticed you had pushed a commit to this PR, but my force-push removed it. Did we do the same update, or was your commit inclusive of more?

Reply:
Not sure, I can look at this tomorrow.